src/javascripts/application.mjs

```diff
@@ -104,3 +111,52 @@ if (campaignCookieBanner) {
     subtree: true
   })
 }
+
+const serviceNavigationHeader = document.querySelector(
+  '.govuk-service-navigation'
+)
```

Comment:
**suggestion** Create a `MobileNavigation` component set up on a `<div>` that wraps the service navigation.